src/components/Body/index.jsx

```diff
@@ -1,62 +1,49 @@
-import React from 'react';
-import { Grid, Typography } from '@mui/material';
-import Card from '../Card';
+import Slider from "react-slick";
+import React, { useEffect, useState } from "react";
+import Cards from "../Card";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import "./styles.css";
+import { getReview } from "../../api/review";
+
+const review = "https://hiddengem.pythonanywhere.com/api/review";
+const users = "https://jsonplaceholder.typicode.com/users";
+
+export default function Body() {
+  const reviewsPerSlide = 3;
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    const reviewFromApi = getReview();
+    reviewFromApi.then((data) => {
+      setReviews(data);
+    });
+  }, []);
+
+  const settings = {
+    infinite: true,
+    speed: 500,
+    slidesToShow: reviewsPerSlide,
+    slidesToScroll: 1,
+    initialSlide: 0,
+  };
 
-const Body = () => {
   return (
-    <Grid data-testid="body" container spacing={3} marginTop={5} marginBottom={13}>
-      {/* Main Content Sections */}
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-        <Typography variant="h1" gutterBottom textAlign={"center"}>
-          New Reviews
```

Comment:
Please add the title back on top of the slider

`New Reviews`

Reply:
udah mbak

Reply:
I don't see your updated changes, can you please push your latest commit?